public/src/components/switchboard.tsx

```diff
@@ -196,24 +365,7 @@ const Switchboard: React.FC<InnerProps<SupportFrontendSwitches>> = ({
 
       {displayNeedToSaveDataWarning()}
       <SaveButton />
-
       <Divider variant="fullWidth" className={classes.divider} />
```

Comment:
We could probably remove the Divider as well, since there’s now nothing below it